js/scenes/Item.js

```diff
@@ -58,10 +59,16 @@ class Item extends Component {
         source={{ uri: imageUri }}
         resizeMode="cover"
         style={styles.image}>
-        <Star
-          value={false}
-          onValueChange={() => {}}
-          style={styles.star}/>
+        <View style={styles.topBar}>
+          <Profile
```

Comment:
Ah I see you're using it here too now which makes sense for it being its own component. But are the designs the same?